app/data/0_rogue_one.json

```diff
@@ -8,13 +8,23 @@
     "description": "Test with a confirmation page instead of a summary",
     "theme": "default",
     "legal_basis": "StatisticsOfTradeAct",
-    "introduction": {
-        "description": ""
-    },
     "groups": [
         {
             "blocks": [
                 {
+                    "type": "introduction",
+                    "id": "introduction",
+                    "title": "Introduction",
+                    "sections": [
```

Comment:
Seems like the addition of empty lists is a workaround for our parser which incorrectly assumes there must be 1 of everything. This is throughout all our other schemas too. Seems like this is what travis is complaining about too.